src/legacy/core_plugins/interpreter/public/functions/tagcloud.js

```diff
@@ -31,13 +31,63 @@ export const tagcloud = () => ({
     defaultMessage: 'Tagcloud visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    scale: {
+      types: ['string'],
+      default: 'linear',
+    },
+    orientation: {
+      types: ['string'],
+      default: 'single',
+    },
+    minFontSize: {
+      types: ['number'],
+      default: 18,
+    },
+    maxFontSize: {
+      types: ['number'],
+      default: 72
+    },
+    showLabel: {
+      types: ['boolean'],
+      default: true,
+    },
+    metric: {
+      types: ['string', 'number'],
     },
+    bucket: {
+      types: ['string', 'number'],
+    },
+    bucketFormat: {
+      types: ['string'],
+      default: 'string'
+    },
+    bucketFormatParams: {
+      types: ['string'],
+      default: '"{}"',
+    }
```

Comment:
The interpreter tagcloud function unit tests will probably need to be updated too